InfiniteRecharge/src/main/java/frc/robot/subsystem/climber/ClimbSubsystem.java

```diff
@@ -55,9 +57,6 @@ public void periodic(float deltaTime) {
             case Extending:
                 motorRight.set(SmartDashboard.getNumber(getName() + "/Climber Current", ClimbConstants.EXTEND_OUTPUT));
                 SmartDashboard.putString(getName() + "/ClimbState", "Extending");
-                if (motorRight.getStatorCurrent() < 1) {
-                    climbState = ClimbState.Off;

```

Comment:
Yeah, I need to talk to Clayton about this. I realized yesterday that this wouldn’t work. He needs to delay checking the current for a loop or two.